dev/com.ibm.ws.jdbc_fat_driver/fat/src/com/ibm/ws/jdbc/fat/driver/JDBCDriverManagerTest.java

```diff
@@ -62,6 +62,7 @@ public static void setUp() throws Exception {
 
     @AfterClass
     public static void tearDown() throws Exception {
-        server.stopServer("DSRA8020E.*internal.nonship.function"); // TODO remove once the capability becomes GA
+        server.stopServer("DSRA8020E.*internal.nonship.function", // TODO remove once the capability becomes GA
+                          "CWWKE0701E"); //Expected in testGetSetLoginTimeout
```

Comment:
I assume this should be removed once ignoreWarnFail logic is replace with always failing

Reply:
No, it won't be because there will still be an injection failure.